File: level2/전광판.py
n = int(input())

number = {'1':"0010010", 
          '2':"1011101", 
          '3':"1011011", 
          '4':"0111010", 
          '5':"1101011",
          '6':"1101111",
          '7':"1110010",
          '8':"1111111",
          '9':"1111011",
          '0':"1110111",
          '_':"0000000"}

for i in range(n):
    cnt = 0
    In, Out = map(str, input().split())
    
    In = '_'*(5-len(In)) + In
    Out = '_'*(5-len(Out)) + Out

    for j in range(5):
        if In[j] != Out[j]:
            for k in range(7):
                if number[In[j]][k] != number[Out[j]][k]:
                    cnt+=1
    print(cnt)

File: level2/전광판.cpp
#include<bits/stdc++.h>
using namespace std;

map<char,string> number = {{'0',"1011111"},
{'1',"0000101"},
{'2',"1110110"},
{'3',"1110101"},
{'4',"0101101"},
{'5',"1111001"},
{'6',"1111011"},
{'7',"1001101"},
{'8',"1111111"},
{'9',"1111101"},
{'_',"0000000"}};

int main(int argc, char** argv)
{
	int T;
	cin >> T;
    
	for (int i=0;i<T;i++){
        string first, second;
		cin >> first >> second;
		int first_blank_cnt = 5-first.size();
        int second_blank_cnt = 5-second.size();
        string blank = "_";
        for (int i = 0;i<first_blank_cnt;i++)
            first = blank + first;
        for (int i = 0;i<second_blank_cnt;i++)
            second = blank + second;
        int cnt = 0;
        for (int i = 0;i<5;i++){
            if (first[i] != second[i]){
                for (int j=0;j<7;j++){
                    if (number[first[i]][j] != number[second[i]][j]){
                        cnt+=1;
                    }
                }
                
            }
        }
        cout << cnt<<endl;

	}
	return 0;
}